RELEASING.md

```diff
@@ -1,4 +1,5 @@
-1. Update the version number in `RCPurchases.m`, `Purchases.podspec` and in `Purchases/Info.plist`
+1. Update the version number in `RCPurchases.m`
```

Comment:
Would be awesome if we didn't have to do both. Difficult to rely on plist presence in 3rd party apps though.

Reply:
I have a plan for that 😉 PR incoming